### ISSUE TITLE
scrollbar bug in shopping cart demo

### ISSUE THREAD

**Issue by bokes** [codef0rmer/angular-dragdrop#341]
If your scrollbar is at the top, no problem, but when you scroll down and then try to drag an item into the cart, like the iphone, the dragged item is way below the mouse position. The erroneous offset is equal to how far the scrollbar is from it's topmost position.
Has anyone fixed this?

**Comment by shanytc** [comment]
yes i have this as well, seems like the plugin doersn't take into affect the scroll position
